earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -147,12 +147,13 @@ sudo yum install -y \
   libcap-devel libmng-devel libpng-devel libX11-devel libXcursor-devel \
   libXft-devel libXinerama-devel libxml2-devel libXmu-devel libXrandr-devel \
   ogdi-devel openjpeg-devel openjpeg2-devel openssl-devel pcre pcre-devel \
-  perl-Alien-Packages perl-Perl4-CoreLibs proj-devel python-devel python27-devel python-unittest2 \
+  perl-Alien-Packages perl-Perl4-CoreLibs proj-devel \
+  python-devel python27-devel python27-pip python-unittest2 \
   rpm-build rpmrebuild rsync scons shunit2 \
   xerces-c xerces-c-devel xorg-x11-server-devel yaml-cpp-devel zlib-devel
```

Comment:
We need another step here to run `sudo pip2.7 install defusedxml` after running the yum command.

Let me summarize everything so we're on the same page:

- We need to install defusedxml for the proper version of python on:
    - All platforms (CentOS 6, 7, and Ubuntu 16)
    - All methods for installing GEEP (manual instructions and RPMs/DEBs)
- For Ubuntu, installing `python-defusedxml` works fine. So we need to:
    - Add `python-defusedxml` to the manual instructions
    - Add `python-defusedxml` as a DEB requirement (so that it will be installed automatically when we install the DEB)
- For CentOS 6, installing `python-defusedxml` will install defusedxml for the wrong version of Python, so we have to fall back to a two-step process: 1) install pip2.7 and 2) use pip2.7 to install defusedxml.
    - For the manual instructions, we simply add both commands.
    - For the RPMs, we can add `python-pip27` as a requirement, but there's no way to add a pip package as a requirement. Thus, we have to do the second step (`pip install defusedxml`) in the pre-install script.
- For CentOS 7, installing `python-defusedxml` works fine, so we can either go with that method or use the same two-step process as on CentOS 6 for consistency.

Hopefully that gives you a clearer picture of what needs to be changed and where.